src/wpublish/QueriesAndChunks.h

```diff
@@ -305,11 +301,15 @@ class QueriesAndChunks {
     std::chrono::seconds _examineAfter = std::chrono::minutes(5);
 
     /// Maximum number of tasks that can be booted until entire UserQuery is put on snailScan.
-    int _maxTasksBooted = 25;
+    int _maxTasksBooted = 15;
```

Comment:
Another magic number w/o a proper explanation.

Reply:
This is set by a configuration file later, so this value is pointless. Adding info to the comment.